templates/books-show.html

```diff
@@ -0,0 +1,47 @@
+<div>
+  <div class="col-md-10 col-md-offset-1">
+	<div class="panel panel-default">
+	  <div class="panel-body">
+
+		<div class='row'>
+		  <div class="col-md-3 col-xs-12 thumbnail">
+		    <img ng-src="{{bookShowController.book.image}}">
+		    <input ng-show="editing" ng-model="bookShowController.book.image" type="text" class="form-control input-md">
+		  </div>
+
+			<div class="col-md-9 col-xs-12">
+			  <ul class="list-group form-inline">
```

Comment:
The amount of indentation changes between 2 spaces and 4 spaces. Try to keep it more consistent.